packages/lodestar-db/src/const.ts

```diff
@@ -0,0 +1,2 @@
+export const BUCKET_LENGTH = 1;
+export const FORK_VERSION_LENGTH = 8;
```

Comment:
Would that mean that all keys in the DB will consume 8 extra bytes? How many keys do we have in the DB usually?

Reply:
Yes, no idea. But we cannot create new bucket on every fork since we have finite amount of buckets (256). 

But even if we have 100k keys, that's 0.8mb db size increase (probably even less since it will be compressed)

Reply:
Alternative:
Increase bucket length to allow for more buckets (eg: increase to 2 or 4)
Create a bucket(+"repository") for every new datatype (eg: Bucket.blockArchive_phase0, Bucket.blockArchive_lightClient)
In this case, the database doesn't need to enforce or understand fork versions. Rather, the enforcement will come from consumer logic which stores to different buckets according to fork-specific implementations.
The database is only responsible for storing different types of data. The node / higher-level logic will be responsible for storing/fetching in a fork-compatible way.
Of course, any wrapper "repositories" can still be done as in this PR.

Reply:
> Increase bucket length to allow for more buckets (eg: increase to 2 or 4)

Sound that with 65536 it plenty of buckets